game/src/test/java/com/hiddenswitch/spellsource/WitchDoctorTests.java

```diff
@@ -3,7 +3,12 @@
 import net.demilich.metastone.game.cards.Attribute;
```

Comment:
Just a heads up, these aren't witch doctor cards, because their `heroClass` is Jade. That makes them monk cards. Unless the hero class isn't supposed to be Monk, and instead is supposed to be `ROSE`, which is Witch Doctor. Please check on it.

Reply:
Hmmm should I put all these tests to custom tests then?